robocup_bringup/thirdparty.repos

```diff
@@ -6,7 +6,7 @@ repositories:
   ThirdParty/small_objects_detector:
     type: git
     url: https://github.com/gentlebots/small_objects_detector.git
```

Comment:
there is a namespace by default the makes navitation to subscribe to a different topic

Reply:
Done